06-building-a-program-start-to-finish/solving-the-problem/output/InvertedIndex.js

```diff
@@ -0,0 +1,126 @@
+/**
```

Comment:
Please fix style violations.
1. Indents should be with two spaces.
2. 80 Characters per line maximum.